gulpfile.js

```diff
@@ -0,0 +1,33 @@
+
+// used by travis-ci to check the hiera json files
+
+var gulp = require("gulp");
+var jsonlint = require("gulp-jsonlint");
+var expect = require('gulp-expect-file');
+var gutil = require('gulp-util');
+
+// custom reporter that should throw exception on lint fail
+var myFailReporter = function (file) {
```

Comment:
Super minor nitpick:

We all try to follow the same styleguide and not have any spaces between `function` and the opening brace.